Sources/PackageGraph/DependencyResolver.swift

```diff
@@ -32,14 +33,14 @@ extension DependencyResolverError: CustomStringConvertible {
 
 public protocol DependencyResolverDelegate {
     func willResolve(term: Term)
-    func didResolve(term: Term, version: Version)
+    func didResolve(term: Term, version: Version, duration: DispatchTimeInterval)
```

Comment:
This is great — one of the clients of libSwiftPM that I know of already has a way to feed durations into the log.